bugsnag-plugin-android-ndk/src/main/java/com/bugsnag/android/ndk/NativeBridge.java

```diff
@@ -31,7 +31,6 @@
     private static final int METADATA_SECTION = 0;
     private static final int METADATA_KEY = 1;
     private static final int METADATA_VALUE = 2;
-    private static final String LOG_TAG = "BugsnagNDK:NativeBridge";
```

Comment:
Worth noting for the changelog that the log tag has changed for NDK logs? Was this intentional?

Reply:
I don't think we have ever documented the log tag before. Having the same log tag should make it somewhat easier to search for all the 'bugsnag' logs - or at least that was the reasoning behind this change.

Reply:
Yeah even though we haven't documented it before, it's possible people may have used it.
I agree that the change is a positive one, I'm just saying we should ensure it goes in the changelog/upgrade guide.

Reply:
Added a note in the upgrading section